test/e2e/operator_test.go

```diff
@@ -0,0 +1,22 @@
+package e2e
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	test "github.com/openshift/cluster-kube-controller-manager-operator/test/library"
+)
+
+func TestOperatorNamespace(t *testing.T) {
+	kubeConfig, err := test.NewClientConfigForTest()
+	require.NoError(t, err)
```

Comment:
such a useless dep.  Who wanted this?  I won't straight out nak, but I haven't seen any real utility.

Reply:
I removed it.